src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,82 @@
+package duke.task;
+
+import duke.DukeException;
+
+public abstract class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a String representing whether the task is done.
+     * Used in the String representation of the task.
+     *
+     * @return String representing whether the task is done.
+     */
+    public String getStatusIcon() {
```

Comment:
I like this way of getting the mark for task done. It is convenient and can be added to string much more easily.